todo-app/src/toDos-DB.js

```diff
@@ -0,0 +1,18 @@
+export let ToDos=[
+    {
+        todoDesc: "Get Out of Bed",
+        toDoStatus: "Done"
+    },
+    {
+        todoDesc: "Brush Teeth",
+        toDoStatus: "Not Done"
+    },
+    {
+        todoDesc: "Eat Breakfast",
+        toDoStatus: "Not Done"
+    }
+]
+
+export default function updateStatus(index, status) {
```

Comment:
You shouldn't use this function here that updates the database directly. The fact that this is a JavaScript file is convenient, but what if this was a JSON file. You wouldn't be able to update the todos. There are other ways to accomplish this in React.

Reply:
hi Spyros ,
 thanks for your feed back, could you please tell me a sample how to do the Update with Ract

Reply:
Hi @aymansalah1! Take a look at my comment at `todo-app/src/ToDoStatus.js`. You can use a combination of `state`, and input's `checked` and `onChange` attributes to accomplish this.